dataset/dictionary/enemies.json5

```diff
@@ -695,29 +810,40 @@
   {
     en: "Electro Whopperflower",
     ja: "トリックフラワー・雷",
+    zh: "电气骗骗花",
     pronunciationJa: "トリックフラワー・かみなり",
     variants: {
       en: [ "Electro Trickflower", "Electro Trick Flower" ],
+      zh: [ "雷骗骗花" ],
     },
     tags: [ "enemy" ],
   },
   {
     en: "Pyro Regisvine",
     ja: "爆炎樹",
+    zh: "爆炎树",
     pronunciationJa: "ばくえんじゅ",
+    variants: {
+      zh: [ "火树" ],
+    },
     tags: [ "enemy-boss", "liyue" ],
   },
   {
     en: "Cryo Regisvine",
     ja: "急凍樹",
+    zh: "急冻树",
     pronunciationJa: "きゅうとうじゅ",
+    variants: {
+      zh: [ "冰树" ],
+    }
```

Comment:
Oops, sorry, you also need to run `npm run build` to eliminate all the errors. Here is also a missing trailing comma.

Reply:
I'm sorry for that. I will try to use `npm run build` next time.

Reply:
It seems that some package need to be installed for me before using it.

Reply:
Yes, you need to run `npm install` or `npm ci` before running `npm run build`. I will write about it when I write README.md.

Reply:
Are there any difficulties to install npm packages in China? I hear JS engineers in China use the Chinese npm mirrors provided by Alibaba or Tencent.
Let me know if I can help you.

Reply:
Thanks, I've solved it and I will ask you to review it after I correct all the errors.